bigbluebutton-html5/imports/ui/components/actions-bar/service.js

```diff
@@ -0,0 +1,19 @@
+import AuthSingleton from '/imports/ui/services/auth/index.js';
+import Users from '/imports/api/users';
+
+let isUserPresenter = () => {

```

Comment:
By the name of this function it should return a `bool` and not an object :smile: